graphql/server.go

```diff
@@ -500,6 +518,73 @@ func CreateJSONSocketWithMutationSchema(ctx context.Context, socket JSONSocket,
 	}
 }
 
+type noopSubscriptionLogger struct{}
+
+func (l *noopSubscriptionLogger) Subscribe(ctx context.Context, id string, tags map[string]string) {}
+func (l *noopSubscriptionLogger) Unsubscribe(ctx context.Context, id string)                       {}
+
+type noopGraphqlLogger struct{}
+
+func (l *noopGraphqlLogger) StartExecution(ctx context.Context, tags map[string]string, initial bool) {
+}
+func (l *noopGraphqlLogger) FinishExecution(ctx context.Context, tags map[string]string, delay time.Duration) {
+}
+func (l *noopGraphqlLogger) Error(ctx context.Context, err error, tags map[string]string) {}
```

Comment:
nso: `nop` instead of `noop` following how `io.NopCloser` is named?

Reply:
done